src/components/Scorecard.vue

```diff
@@ -50,23 +50,21 @@
             :key="metric"
```

Comment:
this is the third comment regarding the UI not matching :/

look at the colors:
- the green color that you have added doesn't match the Figma mockup - did you ask Karn for the color?
- the green color in the progress bar of the scorecard you have added is different from the green color of the button

look at the button sizes, they are not matching too.

look at the relative sizes of the texts.
- in the mockup, the text in the circular progress bar is clearly bigger than the text in the metric boxes
- but in your case, both the texts look the same 


<img width="497" alt="Screenshot 2022-04-18 at 10 38 52 AM" src="https://user-images.githubusercontent.com/13185699/163757878-bb22da02-3cff-4ff3-babb-d48c7153be9d.png">
<img width="638" alt="Screenshot 2022-04-18 at 10 38 56 AM" src="https://user-images.githubusercontent.com/13185699/163757889-ab54ecf1-5fb9-4099-961b-4419f7145dd2.png">